jdaviz/app.py

```diff
@@ -2497,10 +2511,9 @@ def _create_viewer_item(self, viewer, vid=None, name=None, reference=None,
             'name': name or vid,
             'widget': "IPY_MODEL_" + viewer.figure_widget.model_id,
             'toolbar': "IPY_MODEL_" + viewer.toolbar.model_id if viewer.toolbar else '',  # noqa
-            'layer_options': "IPY_MODEL_" + viewer.layer_options.model_id,
-            'viewer_options': "IPY_MODEL_" + viewer.viewer_options.model_id,
+            'data_menu': 'IPY_MODEL_' + viewer._data_menu.model_id if hasattr(viewer, '_data_menu') else '',  # noqa
+            # TODO: remove unused entries after old data menu deprecation period
```

Comment:
What is the thing for "old data menu" that we can grep for to find this again when deprecation period is done?

Reply:
I don't know that there is anything easily greppable, but I added links back to all these comments from the ticket [:cat:](https://jira.stsci.edu/browse/JDAT-4508)